packages/browser-wallet/src/popup/popupX/pages/Accounts/Accounts.scss

```
@@ -25,9 +25,32 @@
             color: $color-white;
         }
 
+        .wrap-anywhere {
+            overflow-wrap: anywhere;
+        }
+
+        input.editable {
+            background: none;
+            color: inherit;
+            border: none;
+            font-weight: inherit;
+            font-size: inherit;
+            font-family: inherit;
+            padding: 0;
+            margin: 0;
+
+            &:focus {
+                outline: none;
+            }
+        }
+
         .button__icon {
             gap: rem(16px);
 
+            svg {
+                width: rem(16px);
+            }
```

```suggestion
            svg {
                width: rem(16px);
                
                &.checkmark-icon {
                    width: rem(12px);
                }
            }
```

Assuming the corresponding class is added to the icon in the view